README.md

```diff
@@ -38,7 +38,7 @@ Phlex::Heroicons.configuration.default_variant = :outline
 
 ## Usage
 ```ruby
-class Hello < Phlex::View
+class Hello < Phlex::HTML
```

Comment:
`Phlex::View` was renamed to `Phlex::HTML` in [phlex@0.5.0](https://github.com/phlex-ruby/phlex/releases/tag/0.5.0).